models/error.go

```diff
@@ -253,6 +253,19 @@ func (err ErrAccessTokenNotExist) Error() string {
 	return fmt.Sprintf("access token does not exist [sha: %s]", err.SHA)
 }
 
+type ErrAccessTokenNameAlreadyExist struct {
```

Comment:
Could you put this new error type in the [errors](https://github.com/gogs/gogs/tree/master/models/errors) subdirectory? We should create a new file named `token.go` to include it.